README.md

```diff
@@ -150,6 +136,16 @@ functions** as [WebAssembly
 imports](https://webassembly.github.io/spec/core/syntax/modules.html#imports)
 (all of them defined in the `oak` module):
 
+-   `wait_on_channels: (i32, i32) -> i32`: Blocks until data is available for
+    reading from one of the specified channel handles.  The channel handles are
+    encoded in a buffer that holds N contiguous 9-byte chunks, each of which is
+    made up of an 8-byte channel handle value (little-endian u64) followed by a
+    single byte that is set on return if data is available on that channel.
+
+    *   arg0: Address of handle status buffer
+    *   arg1: Count N of handles provided
+    *   return 0: Status of operation
```

Comment:
add a link to https://fuchsia.dev/fuchsia-src/zircon/syscalls/object_wait_many.md ? I think it's a useful reference for people who are not really familiar with these concepts.